ibm/service/vpc/resource_ibm_is_vpn_server.go

```diff
@@ -544,7 +544,10 @@ func isWaitForVPNServerStable(context context.Context, sess *vpcv1.VpcV1, d *sch
 			vpnServer, response, err := sess.GetVPNServerWithContext(context, getVPNServerOptions)
 			if err != nil {
 				log.Printf("[DEBUG] GetVPNServerWithContext failed %s\n%s", err, response)
-				return vpnServer, "", fmt.Errorf("Error Getting VPC Server: %s\n%s", err, response)
+				if strings.Contains(err.Error(), "context deadline exceeded") {
+					return vpnServer, isVPNServerStatusPending, nil
```

Comment:
Build is failing here, Can you please check..

<img width="899" alt="Screenshot 2023-05-04 at 9 51 35 AM" src="https://user-images.githubusercontent.com/127872893/236111955-147bded9-3b69-443e-88d4-7abeba979bc3.png">